src/Standards/Generic/Docs/Formatting/SpaceBeforeCastStandard.xml

```diff
@@ -0,0 +1,31 @@
+<documentation title="Space Before Casts">
+    <standard>
+    <![CDATA[
+    Ensures there is exactly one space before a cast operator.
```

Comment:
```suggestion
    There should be exactly one space before a cast operator.
```

Most standard descriptions are quite explicit/directive in the descriptions - same with the error messages.
Have a look at some of the others to see for yourself.